LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/font-style-parsing-expected.txt

```diff
@@ -41,5 +41,5 @@ PASS Font-style (computed): 'oblique' followed by positive angle is valid
 PASS Font-style (computed): 'oblique' followed by calc is valid
 PASS Font-style (computed): 'oblique' followed by calc is valid even if it must be clamped (no computation)
 PASS Font-style (computed): 'oblique' followed by calc is valid even if it must be clamped (with computation)
-PASS Font-style (computed): 'oblique' followed by calc is valid even if it mixes units (with computation)
+FAIL Font-style (computed): 'oblique' followed by calc is valid even if it mixes units (with computation) assert_equals: Font-style computed style: 'oblique' followed by calc is valid even if it mixes units (with computation) expected "oblique 14deg" but got "oblique"
```

Comment:
Happy to do change but will wait for Myles's input as well. :-)

Reply:
The spec [says](http://w3c.github.io/csswg-drafts/css-fonts-4/#font-style-prop):

> Computed value: the keyword specified, plus angle in degrees if specified

I think that "if specified" answers the question. But I do agree that @nt1m has a good point for compatibility. I think we should match the spec as it stands, but also open a CSSWG issue to see if we can remove the "if specified" clause.

Reply:
The test in question is `oblique calc(10grad + 5deg)`, so it is specified. I guess this is more about the "serialize to shortest form" rule here. That becomes `oblique 14deg` and since `14deg` is default it is omitted (which I think shouldn't be the case, for compat reasons)

Reply:
Anyway, my suggested change should fix this.